content/en/docs/application-store/built-in-apps/tomcat-app.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Deploy Tomcat"
+keywords: 'demploy, tomcat'
+description: 'Deploy Tomcat'
+
+linkTitle: "Deploy Tomcat"
+weight: 2110
+---
+
+This tutorial walks you through an example of how to demploy tomcat.
+
+## Prerequisites
+
+- Please make sure you [enable the OpenPitrix system](https://kubesphere.io/docs/pluggable-components/app-store/). Tomcat will be deployed from the App Store.
+- You need to create a workspace, a project, and a user account for this tutorial.  The account needs to be a platform regular user and to be invited as the project operator with the `operator` role. In this tutorial, you log in as `project-operator` and work in the project `test-project` in the workspace `test-workspace`.
+
+## Hands-on Lab
+
+### Step 1: Deploy Tomcat
+
+Please make sure you are landing on the **Overview** page of the project `test-project`.
+
+1. Go to **App Store**.
+
+![go-to-app-store](/images/docs/tomcat-app/tomcat-app01.jpg)
+
+2. Find **Tomcat** and click **Deploy**.
+
+![find-tomcat](/images/docs/tomcat-app/tomcat-app02.jpg)
+
+![click-deploy](/images/docs/tomcat-app/tomcat-app03.jpg)
+
+3. Make sure MySQL is deployed in `test-project` and click **Next**.
+
+![click-next](/images/docs/tomcat-app/tomcat-app04.jpg)
+
+4. Use the default configuration and click **Deploy**.
+
+![click-demploy](/images/docs/tomcat-app/tomcat-app05.jpg)
+
+5. Wait until Tomcat is up and running.
+
+![check-if-tomcat-is-running](/images/docs/tomcat-app/tomcat-app06.jpg)
+
+### Step 2: Access Tomcat Terminal
+
+1. Go to **Services** and click **tomcat-service-name**.
+
+![click-tomcat-service](/images/docs/tomcat-app/tomcat-app07.jpg)
+
+2. Expand pods information and click **terminal**. You can now use the feature.
```

Comment:
From the [preview page](https://deploy-preview-393--kubesphere-v3.netlify.app/docs/application-store/built-in-apps/tomcat-app/#step-2-access-tomcat-terminal), it seems there is a broken image.
![image](https://user-images.githubusercontent.com/40452856/97319046-76ff9b00-18a7-11eb-9424-f8d06a71390c.png)